docs/manual/docs/cmd/graph/teams/teams-user-add.md

```diff
@@ -0,0 +1,42 @@
+# graph teams user add
+
+Adds an owner or member to the specified Microsoft Teams team
+
+## Usage
+
+```sh
+graph teams user add [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --teamId <teamId>`|The ID of the team for which to list users
```

Comment:
Incorrect. Should be `The ID of the team for which to add a new owner or member`